pkg/aws/services/ec2.go

```diff
@@ -112,7 +126,11 @@ func (c *ec2Client) DescribeSubnetsAsList(ctx context.Context, input *ec2.Descri
 
 func (c *ec2Client) DescribeVPCsAsList(ctx context.Context, input *ec2.DescribeVpcsInput) ([]types.Vpc, error) {
 	var result []types.Vpc
-	paginator := ec2.NewDescribeVpcsPaginator(c.ec2Client, input)
+	client, err := c.awsClientsProvider.GetEC2Client(ctx, "DescribeVPCs")
```

Comment:
for the passed-in "operationName", let's use the real AWS operation name. You can find it in the generated SDK:
https://github.com/aws/aws-sdk-go-v2/blob/f8631c3fcf531450411deb514e5d5a18e93824f0/service/ec2/api_op_DescribeVpcs.go#L27
e.g. it shall be `DescribeVpcs` instead of `DescribeVPCs`

Reply:
fixed